modules/tensor_mechanics/test/tests/beam/dynamic/tests

```diff
@@ -1,173 +1,287 @@
+
+    requirement = "The dynamic system shall correctly predict the natural frequencies"
+                  " of an Euler-Bernoulli beam in the presence of Rayleigh damping"
+                  " and numerical damping induced by HHT time integration."
   [../]
   [./dyn_euler_added_mass]
     type = 'Exodiff'
     input = 'dyn_euler_small_added_mass.i'
     exodiff = 'dyn_euler_small_added_mass_out.e'
+
+    requirement = "The dynamic system shall correctly predict the natural frequencies"
+                  " of a massless Euler-Bernoulli beam with a nodal masses placed"
+                  " at the ends."
   [../]
   [./dyn_euler_added_mass_file]
     type = 'Exodiff'
     input = 'dyn_euler_small_added_mass_file.i'
     cli_args = "NodalKernels/x_inertial/nodal_mass_file=nodal_mass.csv"
     exodiff = 'dyn_euler_small_added_mass_out.e'
     prereq = 'dyn_euler_added_mass'
+
+    requirement = "The dynamic system shall correctly predict the natural frequencies"
+                  " of a massless Euler-Bernoulli beam with added nodal masses when"
+                  " the location and values of the masses are provided using a csv file."
   [../]
   [./dyn_euler_added_mass_gravity]
     type = 'Exodiff'
     input = 'dyn_euler_small_added_mass_gravity.i'
     cli_args = "NodalKernels/force_y2/mass=0.01899772"
     exodiff = 'dyn_euler_small_added_mass_out.e'
     prereq = 'dyn_euler_added_mass_file'
+
+    requirement = "The dynamic system shall correctly model the beam's response"
+                  " under gravitational force (proportional to nodal mass) is"
+                  " applied to the beam."
   [../]
   [./dyn_euler_added_mass_gravity_2]
     type = 'Exodiff'
     input = 'dyn_euler_small_added_mass_gravity.i'
     cli_args = "NodalKernels/force_y2/nodal_mass_file=nodal_mass.csv"
     exodiff = 'dyn_euler_small_added_mass_out.e'
     prereq = 'dyn_euler_added_mass_gravity'
+
+    requirement = "The dynamic system shall correctly model the beam's response"
+                  " under gravitational force when the nodal mass distribution is"
+                  " provided using a csv file."
   [../]
   [./add_dynamic_variables_action]
     type = 'Exodiff'
     input = 'dyn_euler_small_added_mass_dyn_variable_action.i'
     exodiff = 'dyn_euler_small_added_mass_out.e'
     prereq = 'dyn_euler_added_mass_gravity_2'
+
+    requirement = "The LineElementAction shall create the translational & rotational"
+                  " velocities and accelerations required for a dynamic beam simulation."
+    design = "C0TimoshenkoBeam.md LineElementMaster/index.md"
   [../]
   [./dyn_euler_added_mass_inertia_damping]
     type = 'Exodiff'
     input = 'dyn_euler_small_added_mass_inertia_damping.i'
     exodiff = 'dyn_euler_small_added_mass_inertia_damping_out.e'
+
+    requirement = "The dynamic system shall correctly model the beam's response"
+                  " in the presence of nodal mass, nodal inertia and Rayleigh damping."
```

Comment:
dynamic -> mechanics, 'the beam's response' -> 'the response of a beam modeled using beam elements'